src/syscheckd/src/db/include/db.h

```diff
@@ -187,50 +195,6 @@ void fim_db_teardown();
 
 // Registry functions.
 
-/**
- * @brief Get registry data using its key_id and name.
- *
- * @param key_id ID of the registry.
- * @param name Name of the registry value.
- *
- * @return FIM registry data struct on success, NULL on error.
- */
-fim_registry_value_data* fim_db_get_registry_data(unsigned int key_id, const char* name);
```

Comment:
If all of the removed declarations are deprecated functions that no longer exist. It would be great to also remove their references in the unit tests. In particular, at https://github.com/wazuh/wazuh/tree/14432-update-design-graphs/src/unit_tests/wrappers/wazuh/syscheckd/db_old_wrappers.

Reply:
Done